BeeKit/Model/DataPoint.swift

```diff
@@ -103,6 +107,7 @@ extension DataPoint {
     /// DataPoints are used to track certain events, like automatic pessimistic values, goal restarts, derailments, etc. These should sometimes
     /// be treated differently, e.g. not deleted as part of syncing with HealthKit
     public var isMeta: Bool {
-        DataPoint.metaPointHashtags.contains { comment.contains($0) }
+        guard origin == "beeminder" else { return false }
+        return DataPoint.metaPointHashtags.contains { comment.contains($0) }
```

Comment:
I am thinking checking origin is enough and the comment content check can be skipped. The website is origin beeminderweb or similar.